frontend/src/modules/projectinfo/components/ProjectInfo.js

```diff
@@ -54,11 +55,21 @@ export class ProjectInfoBase extends React.Component<Props, State> {
 
         return <div className="project-info">
             <div className="button" onClick={ this.toggleVisibility }>
-                <span className="fa fa-info"></span>
+              <Localized
+                  id="projectinfo-ProjectInfo--project-info"
+                  glyph={
+                      <span className="fa fa-info"></span>
+                  }
+              >
+              </Localized>
             </div>
             { !this.state.visible ? null :
             <aside className="panel">
-                <h2>Project Info</h2>
+              <Localized
+                  id="projectinfo-ProjectInfo--project-info-title"
+              >
+                  <h2>Project Info</h2>
+              </Localized>
```

Comment:
The indentation here is off. We use 4-space indentation in our front-end, please make sure to configure your editor correctly! :)